### PR TITLE
add new php libraries

### PR DESCRIPTION
**Pull request by portey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,8 @@ If you want to contribute to this list (please do), send me a pull request.
 * [graphql-mapper](https://github.com/4rthem/graphql-mapper) - This library allows to build a GraphQL schema based on your model.
 * [graphql-bundle](https://github.com/suribit/GraphQLBundle) - GraphQL Bundle for Symfony 2.
 * [overblog/graphql-bundle](https://github.com/overblog/GraphQLBundle) - This bundle provides tools to build a complete GraphQL server in your Symfony App. Supports react-relay.
+* [youshido/graphql](https://github.com/Youshido/GraphQL) - GraphQL implementation in PHP using PHP ecosystem
+* [youshido/graphql-bundle](https://github.com/Youshido/GraphQL) - Symfony2/3 bundle for youshido/graphql implementation
```
Comment (sogko) [inline README.md]: I think you meant to put the URL for `youshido/graphql-bundle` as  https://github.com/Youshido/GraphQLBundle

```
* [youshido/graphql-bundle](https://github.com/Youshido/GraphQLBundle) - Symfony2/3 bundle for youshido/graphql implementation
```

Cheers!

**Comment by portey** [comment]
@sogko , yes, I have already fixed it.

**Comment by sogko** [comment]
Awesome, now to wait for @chentsulin to spare his time to merge this ;)

Cheers!

**Comment by chentsulin** [comment]
I'd like to see those repos have detailed README to elaborate why they are awesome.

**Comment by chentsulin** [comment]
duplicate of #70

**Pull request closed without merging**